tracker/templates/challenges.json.j2

```diff
@@ -0,0 +1,49 @@
+    {
+      "name": "append-no-conflicts",
+      "description": "Indexes the whole document corpus using Elasticsearch default settings. We only adjust the number of replicas as we benchmark a single node cluster and Rally will only start the benchmark if the cluster turns green. Document ids are unique so all index operations are append only. After that a couple of queries are run.",
```

Comment:
The description does not match what's going on.

Reply:
https://github.com/elastic/rally/pull/875/commits/7d47ae90c56a03726cb7fccdeec21c2b7b36f171